src/app/beer_garden/api/http/__init__.py

```diff
@@ -332,7 +336,10 @@ def _load_swagger(url_specs, title=None):
             {"$ref": "#/definitions/IntervalTrigger"},
         ]
     }
-    api_spec._definitions["Job"]["properties"]["trigger"] = trigger_properties
+    api_spec._definitions["Job"]["properties"]["trigger"] = trigger_properties  # noqa
+
+    api_spec.definition("JobExport", schema=JobExportSchema)
+    api_spec.definition("JobImport", schema=JobExportListSchema)
```

Comment:
This was much tougher than I expected. The difficulty is that we're using a version of the library that's sub-1.0, whereas the current version is above 5.0. I'm creating a ticket to update our library so that we can at least have the ability to find documentation for what we're trying to do.

At any rate, I believe I've corrected the issues with swagger. I've examined the created web page and it works as expected for both import and export and the schemata it shows are as we expect. Please verify you see the same.